docs/contributing.md

```diff
@@ -20,83 +20,5 @@ Please follow it in all your interactions with the project members and users.
 6. If necessary, update the README.md.
 7. The pull request will be reviewed by others, and the final merge must be done by an OWNER.
 
+For contributing to documentation, see our [Developer Guide](https://oras-project.github.io/oras-py/getting_started/developer-guide.html#documentation).
```

Comment:
Is it better to use a relative link?

Reply:
Only if we need to move it. I think this originally started in the docs that render to the different domain, so I used the full url.